test/Microsoft.AspNetCore.SignalR.Tests/HubEndpointTestUtils/Hubs.cs

```diff
@@ -558,9 +465,11 @@ public ChannelReader<string> BlockingStream()
             return Channel.CreateUnbounded<string>().Reader;
         }
 
-        public IObservable<int> ThrowStream()
+        public ChannelReader<int> ThrowStream()
         {
-            return Observable.Throw<int>(new Exception("Exception from observable"));
+            var channel = Channel.CreateUnbounded<int>();
+            channel.Writer.TryComplete(new Exception("Exception from channel"));
+            return channel.Reader;
         }
 
         private class CountingObservable : IObservable<string>
```

Comment:
Still used? It's worth doing a search for usings of namespace or references to the package in project files that are still floating around.